packages/web-runtime/src/pages/notFound.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <div class="oc-flex oc-flex-center oc-flex-middle oc-flex-column page-not-found">
+    <oc-icon name="emotion-normal" fill-type="line" size="xxlarge" />
+    <h1 class="oc-text-muted">404</h1>
+    <p
+      class="oc-text-xlarge oc-m-rm oc-text-muted"
+      v-text="$gettext('The page you are looking for does not exist.')"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { $gettext } from '@ownclouders/web-pkg/src/router/utils'
```

Comment:
The import path looks weird, something your IDE did automatically? The import shouldn't be necessary at all, no?

Reply:
yes over and over again, intelliJ is doing this